src/web/lib/braze/checkBrazeDependencies.ts

```diff
@@ -0,0 +1,63 @@
+import {
+	checkDependencies,
+	DependencyConfig,
+	DependencyResult,
+} from '@root/src/web/lib/dependencyChecker';
+import { getBrazeUuid } from '@root/src/web/lib/getBrazeUuid';
+import { hasRequiredConsents } from './hasRequiredConsents';
+import { hideSupportMessaging } from './hideSupportMessaging';
+
+const getCompleteDependencies = async (
+	isSignedIn: boolean,
+	idApiUrl: string,
+): Promise<Array<DependencyConfig>> => {
+	return [
+		{
```

Comment:
I suspect prettier is not being applied correctly here.

Reply:
Yeah I wondered the same thing, and had a conversation with Alex S on Friday about it. I _think_ it's just GitHub rendering the diff weirdly. Locally it looks fine and I see the correct number of tab characters. I'll check again to make sure.